templates/internet-of-things/index.html

```diff
@@ -356,6 +356,23 @@ <h3 class="p-card__title">Qualcomm Dragonboard</h3>
       <p class="p-card__content">64-bit ARM boards bring high-end performance to the low-power segment, enabling a new class of drones and mobile intelligence.</p>
       <p class="p-card__content"><a href="https://developer.qualcomm.com/hardware/dragonboard-410c" class="p-link--external">Learn more on qualcomm.com</a></p>
     </div>
+    <div class="col-6 p-card">
+      <header class="no-border u-align--center u-vertically-center u-hide--small" style="min-height: 7rem;">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/22011fee-Xilinx_logo.svg.png",
+          alt="",
+          width="256",
+          height="52",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </header>
+      <hr class="u-sv1 u-hide--small" />
+      <h3 class="p-card__title">Xilinx Evaluation Kits and SOMs</h3>
+      <p class="p-card__content">Zynq&reg; UltraScale+&trade; MPSoCs integrate a quad-core 64-bit Arm application processor with real-time processors and FPGA logic to offer an embedded, low-latency acceleration platform.</p>
+      <p class="p-card__content"><a href="https://www.xilinx.com/applications/industrial/robotics.html#solutionStack?utm_source=canonical&utm_medium=referral&utm_campaign=ubuntu&utm_content=iot&utm_term=eval_kit" class="p-link--external">Learn more on xilinx.com</a></p>
```

Comment:
On the copy doc, only "xilinx.com" is linked. 
```suggestion
      <p class="p-card__content">Learn more on <a href="https://www.xilinx.com/applications/industrial/robotics.html#solutionStack?utm_source=canonical&utm_medium=referral&utm_campaign=ubuntu&utm_content=iot&utm_term=eval_kit" class="p-link--external">xilinx.com</a></p>
```

Reply:
Good spot Min! I actually off my own back included the "learn more on " to be consistent with the other cards but I should have updated the copy doc!